canonical-websites/www.ubuntu.com
# Update /esm page

## Done

- Update the /esm page, mostly the FAQ section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/esm](http://0.0.0.0:8001/esm)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1BK-XXECYJJllG8jIe3WN3X8EMo_3wBBa03nlzYDP8xE/edit#)

## Issue / Card

Fixes #4794
